templates/leaderboard/leaderboard.html

```diff
@@ -10,8 +10,27 @@
 
 {% block content %}
     <h2>Top Investors</h2>
-    <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+
+    <body onload="leaderLoadFunction()" style="margin:0;">
+
+    <div id="loader"></div>
+
+    <div style="display:none;" id="leader-loaded" class="animate-bottom">
+        <div class="center">
+            {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+        </div>
     </div>
 
+<script>
```

Comment:
We should create an `extrajs` block for the JS here. We can use `$(document).ready()` instead of `onload` above, and use jQuery to simplify a few things:
```html
        ...
    </div>
{% endblock %}

{% block extrajs %}
    $(document).ready(function() {
        'use strict'

        function showPage() {
            $('#loader').hide();
            ${'#leader-loaded').css("display", "block");
        }

        setTimeout(showPage, window.onload);
    });
{% endblock %}
```

I didn't test that code out, but I think it will work.

Reply:
So I tried re-doing this using jQuery but having a bit of trouble. Would I need to replace both 'onload' with '$(document).ready()' and 'leaderLoadFunction()'  with 'showPage()'? 

I'm not very familiar with jQuery

Reply:
Oh you know what? I forgot the `<script>` tags. The whole thing should look like this:
```html
        ...
    </div>
{% endblock %}

{% block extrajs %}
    <script type="text/javascript">
        $(document).ready(function() {
            'use strict'

            function showPage() {
                $('#loader').hide();
                ${'#leader-loaded').css("display", "block");
            }

            setTimeout(showPage, window.onload);
        });
    </script>
{% endblock %}
```